packages/playground/src/dashboard/components/user_farms.vue

```diff
@@ -7,6 +7,9 @@
       single-line
       hide-details
     ></v-text-field>
+    <v-alert type="info" variant="tonal" v-if="farmsCount == 0">
```

Comment:
what do u think about styles should we add margin bottom?
@amiraabouhadid @ehab-hassan

Reply:
yes let's add some margin-bottom and I think a warning color here like this
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/10484295/0c0fe217-e002-47e7-9570-757a88d465e0)

Reply:
done, thanks